doc/api.md

```diff
@@ -742,6 +744,58 @@ Unreserves a path for a publishing application. Returns success or failure only.
 - If a path reservation exists for the supplied base_path but a different
   publishing_app, the command will fail.
 
+## `PUT /publish-intent(/*base_path)`
+
+[Request/Response detail][put-intent-pact]
+
+Used to create or update a publishing intent for a particular base_path and
+reserve the base path in the publishing-api. A publishing intent sets when the
+cache should expire for a piece of content.
+
+### Path parameters
+
+- `base_path`
+  - Identifies the path to create the publishing intent for
+
+### JSON parameters:
+
+- `publishing_app` *(required)*
+  - The name of the application making this request.
+- `rendering_app` *(required)*
+  - The name of the application rendering the content the base path refers to.
+- `publish_time` *(required)*
+  - The `DateTime` the content is set to be published.
+- `routes` *(optional)*
+  - `Array` - Fields supported are path and type, the exact requirements can be
+    found [here][routes-content-schema]. This is optional as it is only
+    required if there needs to be a route created in router-api for the
+    base_path.
+
+### State changes
+
+- `base_path` is reserved for use of the given `publishing_app`.
+- Creates a `PublishingIntent` in content-store to record when the cache should
+  expire for a `base_path`.
+- If `routes` are given, adds a route set in router-api.
+
+## `DELETE /publish-intent(/*base_path)`
+
+Used to delete the publishing intent for a particular `base_path`. Removing a
+publishing intent would mean that the cache for the `base_path` would no longer
```

Comment:
The phrasing is a little strange here - the use of 'would' makes it sound like it's conditional on something. Also, the use of 'the cache' is a little misleading, as there many caches (known and unknown); it would be correct to say 'means the content will continue to be cached as normal'.

Reply:
Yep i tried to convey that in this sentence 'would no longer have a specific time in which it should be cleared by.` but if that's not clear enough I'll amend it to your suggestion :)